packages/flutter_tools/templates/module/ios/library/Flutter.tmpl/podhelper.rb.tmpl

```diff
@@ -35,7 +42,7 @@ def install_flutter_engine_pod
   # Keep pod path relative so it can be checked into Podfile.lock.
   # Process will be run from project directory.
   engine_pathname = Pathname.new engine_dir
-  project_directory_pathname = Pathname.new Dir.pwd
+  project_directory_pathname = defined_in_file.dirname
```

Comment:
what is `defined_in_file`? is this a pod magic function?

Reply:
nvm, I grepped the codebase and found y our other comments about it

Reply:
I copied the comments to this file.